frontend/src/utils/api.js

```diff
@@ -104,7 +104,11 @@ function getPastInterviews(interviewerKey: string) {
     res.json()
   )
 }
-
+function getCandidateInterviews(candidateId: string) {
+  return fetch(`${API_URL}/interview/candidate-interviews/${candidateId}?key=${getKey()}`).then(
```

Comment:
the right way would be to get an endpoint that would be like:
`/candidates/:candidateId/interviews`

but this will do for now. No need to change.